int32[] id_order
float32[] obj1_pos
float32[] obj2_pos
float32[] obj3_pos
float32[] obj4_pos
float32[] obj5_pos
float32[] obj6_pos
float32[] obj7_pos
